tests/python/gaia-ui-tests/gaiatest/apps/gallery/regions/view_image.py

```diff
@@ -40,7 +39,7 @@ def tap_back_button(self):
         header.tap(x=20)
 
         # wait for the frame to close
-        Wait(self.marionette).until(lambda m: self.apps.displayed_app.name != 'Gallery')
+        slef.apps.wait_to_not_be_displayed()
```

Comment:
You still need to change this. And I think this needs to be Gallery(self.marionette).wait_to_not_be_displayed(), actually.